style.css

```diff
@@ -0,0 +1,306 @@
+  color: #172b4d;
+}
+
+h3 {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+h3 span {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #7a869a;
+}
+
+h5 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  color: #172b4d;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 30px auto;
+}
+
+.canopy-card {
+  background: #f7f7f9;
+  padding: 20px 0;
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  grid-auto-rows: auto;
+  grid-gap: 1rem;
+}
+
+.snapshoot-portofolio {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.canopy-card-section {
+  display: flex;
+  background: #fff;
+  height: 580px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-second-card-section {
+  background: #fff;
+  height: 610px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-third-card-section {
+  background: #fff;
+  height: 573px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-fourth-card-section {
+  background: #fff;
+  height: 610px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.project-card {
+  width: 100%;
+  margin: -5px auto;
+}
+
+.project-card p {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+.project-card li {
+  display: inline-block;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03;
+  color: #6070ff;
+  background: #ebebff;
+  margin-right: 5px;
+  padding: 7px;
+  list-style: none;
+  border-radius: 8px;
+}
+
+.project-card ul {
+  padding: 0;
+}
+
+button {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03;
+  color: #396df2;
+  border: 1px solid #396df2;
+  padding-right: 30px;
+  margin: 16px auto;
+  border-radius: 8px;
+  background: #fff;
+  cursor: pointer;
+  display: block;
+  position: absolute;
+}
+
+.project-card button {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03;
+  color: #396df2;
+  border: 1px solid #396df2;
+  padding: 7px 16px;
+  margin: 16px auto;
+  border-radius: 8px;
+  background: #fff;
+  cursor: pointer;
+  flex-wrap: wrap;
+}
+
+.skill p {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  color: #344563;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 20px auto;
+}
```

Comment:
- [ ] KIndly make sure this paragraph is aligned well with other content on the page in order to match the Figma design as much as possible.\

  - ### YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/219609801-93285030-a494-4e13-86e5-b6eeaa7b2133.png)

  - ### FIGMA DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/219609909-9b5d5fcf-7017-4b71-92a0-0ff3eab2f371.png)